Specs/Number/Chinese/NumberModel.json

```diff
@@ -4788,5 +4788,65 @@
     "Input": "这是我的队伍",
     "NotSupported": "python, javascript, java",
     "Results": []
+  },
+  {
+    "Input": "小明每天收拾房间",
```

Comment:
Aren't these supported in the other platforms once the resources are generated? If so, please include the updates resource files and enable the tests, ok?
Thanks.